README.md

```diff
@@ -344,14 +344,16 @@ There are a growing number of tools that are built atop or support Zod natively!
 
 - TypeScript 4.1+!
 - You must enable `strict` mode in your `tsconfig.json`. This is a best practice for all TypeScript projects.
+- Make sure that `strictNullChecks` mode is also enabled, otherwise static typechecking will return all properties as optional!
```

Comment:
I think we should highlight the "override" behavior here. Maybe something like:

```suggestion
- Make sure that `strictNullChecks` mode has not been disabled explicitly by setting it to `false` elsewhere, otherwise TypeScript will infer all properties as optional. If you're extending a base config, you may need to set this to be explicitly `true`.
```